sdk/iot/doc/how_to_iot_hub_samples_windows.md

```diff
@@ -0,0 +1,408 @@
+# How to setup and run Azure SDK for Embedded C IoT Hub Samples on Microsoft Windows
+
+_Last edited on June 3rd, 2020._
+
+This is a step-by-step documentation of how to start from scratch and get the Azure SDK for Embedded C IoT Hub Samples running on Microsoft Windows. 
+
+Pre-requisites:
+- [Having created an Azure account](https://github.com/ewertons/azure-sdk-for-c/wiki/How-to-create-an-Azure-account)
+- [Having created an Azure IoT Hub](https://github.com/ewertons/azure-sdk-for-c/wiki/How-to-create-an-Azure-IoT-Hub)
+- Microsoft Visual Studio installed in the local machine.
+
+What is covered:
+- Downloading and building the Azure SDK for Embedded C suite
+- Configuring and running the IoT Hub client samples.
+
+_Run on Microsoft Windows 10.0.18363.836._
+
+01. Install git
+
+  Get the installer from the official git [page](https://git-scm.com/download/win)
+
```

Comment:
So fun fact with README docs. If you want to continue numbered lists but maintain a break between lines of text, you can do it by including two spaces on the line that you want to break and then just continue writing on the next line down. It will align the text to the number. Something like:
```suggestion
01. Install git  
Get the installer from the official git [page](https://git-scm.com/download/win)

```

Reply:
If you could do that here and elsewhere, the alignment might look a little tighter.